packages/components/app/styles/components/dialog.scss

```diff
@@ -0,0 +1,123 @@
+//
+// DIALOG
+//
+
+@use "../mixins/focus-ring" as *;
+
+.hds-dialog {
+  flex-direction: column;
+  padding: 0;
+  background: var(--token-color-surface-primary);
+  border: 1px solid rgba(59, 61, 69, 40%); // --token-color-border-strong;
+  border-radius: 10px;
+  box-shadow: var(--token-elevation-higher-box-shadow);
+
+  &[open] {
```

Comment:
I have looked at the code, but I don't understand how this works: it's not `display: none` by default, nothing is setting an `open` attribute on the `<dialog>` element, so... I'm a bit lost 😅

Reply:
In short, a `<dialog>` is `display:none` by default (I think what the UAs do is actually an exclusion `dialog:not([open]) { display: none; }`. The native `showModal` method (that we invoke in our code) adds the `open` attribute and the native `close` removes it. That's the complete `<dialog>` API in a gist.